static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/design/centesimus.js

```diff
@@ -91,6 +92,10 @@ export const designShow: ({
                     articleBody.style.display = 'block';
                     shadowArticleBody.remove();
 
+                    // The page does not reload when a user dismisses the gate,
+                    // so we must reset page targeting params dynamically via googletags
+                    setGatePageTargeting(false);
```

Comment:
i think even when a user dismisses a gate, the `setGatePageTargeting` should not be changed to `false`, since a gate used to exist on the page, which may have had some impact, so it's worth knowing

Reply:
would a 3rd option to `setGatePageTargeting` solve this, for example we already set `t` or `f`, so is it possible to add something like a `d` for seen and dismissed? or does the ad targeting only let you set true or false?

Reply:
I can add a third value.  I set this to false because there are additional ad requests made on the page after the gate is dismissed, but as at this point the gate is not longer visible on the page. The ad requests are made when ever an ad slot is loaded or refreshed. 

I will ask what is preferred by the commercial analyst

Reply:
thanks, that's great, I'll approve the PR, and we can make changes when they get back to you :)